### DISCUSSION TITLE
Server requests are responding with an error.

### DISCUSSION THREAD

**Discussion by sagarbhosale** [vedicscriptures/bhagavad-gita-api#10] [Q&A] [answered]
**Describe the bug**
All requests are failing.

**To Reproduce**
```
curl https://vedicscripturesapi.herokuapp.com/gita
<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<title>Error</title>
</head>
<body>
<pre>Cannot GET /gita</pre>
</body>
</html>
```

**Additional context**
It was working until yesterday.

**Comment by PtPrashantTripathi** [comment, marked as answer]
> @sagarbhosale kindly refer to the documentation 
https://docs.bhagavadgitaapi.in/api-reference/authentication

**Comment by PtPrashantTripathi** [comment]
Hi @sagarbhosale  did you got my mail
I have sent you api key
From : vedic.scriptures@outlook.com
P.s. kindly Check in spem folder also

**Comment by Nishant0073** [comment]
I am getting the following error while making API call

> Application error
> An error occurred in the application and your page could not be served. If you are the application owner, check your logs for details. You can do this from the Heroku CLI with the command
> heroku logs --tail

**Reply by PtPrashantTripathi** [reply]
Can you please send me screenshots.

**Reply by PtPrashantTripathi** [reply]
hi @Nishant0073 check your mail
